store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -1659,11 +1659,11 @@ private boolean execute(String args[]) throws ServiceException, ArgException, IO
     }
 
     private void doSendMdmEmail(String[] args)  throws ServiceException {
-            if (args.length == 2 && StringUtil.isNullOrEmpty(args[1]) && StringUtil.isNullOrEmpty(args[1])) {
+            if (args.length >= 3 && !StringUtil.isNullOrEmpty(args[1]) && !StringUtil.isNullOrEmpty(args[2])) {
                 prov.sendMdmEmail(args[1], args[2]);
             } else {
-                throw ServiceException.FAILURE(String.format("Number of arguments passed i.e. %s. Status of device : %s. Time interval of email %s. ",
-                        args.length , args[0], args[1]), null);
+                throw ServiceException.FAILURE(String.format("Command execured : %s. Number of arguments passed i.e. %s. Status of device : %s. Time interval of email %s. ",
+                        args[0], args.length , args[1], args[2]), null);
```

Comment:
Typo and out of bounds exception in this throw.

Reply:
resolved